### DISCUSSION TITLE
Clickbait Remover: should it be implemented?

### DISCUSSION THREAD

**Discussion by thanhminhmr** [polymorphicshade/NewPipe#289] [Ideas]
I'm currently using [Clickbait Remover for Youtube](https://addons.mozilla.org/en-GB/firefox/addon/clickbait-remover-for-youtube/). To quote from its desciption: "This extension replaces thumbnails with a frame from the video, effectively removing any clickbait while still showing a high quality thumbnail so you can still get a good idea of what the video is about. It can also modify titles to stop partial or all caps titles."

This is an example of some videos with CR on and off for comparison:
Before | After
|:---:|:---:|
![Linus Tech Tips - Before](https://user-images.githubusercontent.com/31412085/204120327-91f2ad03-dada-4b2b-83d1-ccb115157b8f.png) | ![Linus Tech Tips - After](https://user-images.githubusercontent.com/31412085/204120350-c3ba0c7d-ab39-4d98-9bb2-b80cc1ebc5d2.png)
![GothamChess - Before](https://user-images.githubusercontent.com/31412085/204120328-4c406972-939d-4ee8-8997-a6b2d63e71b7.png) | ![GothamChess - After](https://user-images.githubusercontent.com/31412085/204120329-4a2cb9a4-1a68-42de-ac28-7c41479b9ee5.png)

Should this be a part of this fork? I guess this would never make a way into the upstream with the similar reason to SponsorBlock one.

**Comment by polymorphicshade** [comment]
See #319
